### DISCUSSION TITLE
The Documentation Says There Is  A  "Sleep Functinon"...Which One Is It? Deep, or Light?

### DISCUSSION THREAD

**Discussion by madmacks59** [sandeepmistry/arduino-LoRa#673] [Q&A]
I have several sensors attached to my Heltec ESP32LoRa module (OLED) and before I put the thing to Sleep I'd like to know which Sleep is being used? The manufactures spec sheets talk about Deep Sleep, which shuts down most features on the system, and Light Sleep, which is not a disruptive (more like a clean delay loop) and still save a lot of power. In this library which Sleep is being implemented when the function  is used?

**Comment by morganrallen** [comment]
Unsure what spec sheet you're referring to, the SX1276 datasheet makes no mention of deep sleep vs light speed.

The two operating modes are Standby Mode (activated by the improperly named `LoRa.idle()`) and Sleep Mode (activated by `LoRa.sleep()`).

I suggested reading the [datasheet](https://cdn-shop.adafruit.com/product-files/3179/sx1276_77_78_79.pdf) for more info, but basically you can treat `.idle()` as light and `.sleep()` as deep, the later having a longer startup time.
